lib/node_modules/@stdlib/plot/unicode/static_plots/bar/props/barColor/set.js

```diff
@@ -0,0 +1,51 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var format = require( '@stdlib/string/format' );
+
+
+// VARIABLES //
+
+var colors = require( '@stdlib/plot/unicode/static_plots/marker/colors.js' );
```

Comment:
Don't reach into packages like this. Find another way. If, in the future, we rename the `colors.js` file, this will break here, but we cannot possibly know that beforehand because you pierced the package wall. Files within a package should always be considered private.